### ISSUE TITLE
[Suggestion] Extend SaltItemMelting functionality to items in the salt tag.

### ISSUE THREAD

**Issue by CraftyZombie** [mortuusars/Salt#23]
I love this mod, but I do prefer to use spelunkery's salt since it's the most versatile and can also be placed. Would you consider extending the melting ability to the salt tag to that spelunkery's salt item could also melt blocks?

**Comment by mortuusars** [comment]
Yeah, I should have implemented it differently to allow for other items. 
I'm currently busy with other projects, but I'll try to implement this next time I work on the mod.
